x-pack/plugins/siem/public/components/links/index.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { encodeIpv6 } from '../../lib/helpers';
+
+// Internal Links
+export const HostDetailsLink = pure<{ children?: React.ReactNode; hostName: string }>(
+  ({ children, hostName }) => (
+    <EuiLink href={`#/link-to/hosts/${encodeURIComponent(hostName)}`}>
+      {children ? children : hostName}
+    </EuiLink>
+  )
+);
+
+export const IPDetailsLink = pure<{ children?: React.ReactNode; ip: string }>(
+  ({ children, ip }) => (
+    <EuiLink href={`#/link-to/network/ip/${encodeURIComponent(encodeIpv6(ip))}`}>
+      {children ? children : ip}
+    </EuiLink>
+  )
+);
+
+// External Links
+export const GoogleLink = pure<{ children?: React.ReactNode; link: string }>(
+  ({ children, link }) => (
+    <EuiLink href={`https://www.google.com/search?q=${encodeURI(link)}`} target="_blank">
```

Comment:
Since we use `EuiLink` it looks like our bases are already covered where they insert it for us:

https://github.com/elastic/eui/blob/master/src/services/security/get_secure_rel_for_target.ts#L31
https://github.com/elastic/eui/blob/master/src/components/link/link.js#L46

Hand validated by opening our window and checking the links as well for EuiLinks:

<img width="639" alt="Screen Shot 2019-04-29 at 4 23 02 PM" src="https://user-images.githubusercontent.com/1151048/56930759-16698500-6a9b-11e9-9c77-53ed167e35c7.png">

And I think we are good but thanks for the link and information. Good read.